src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,31 @@
+package duke.command;
+
+import duke.storage.Storage;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import duke.ui.Ui;
```

Comment:
Perhaps it's better to group these together for better consistency across different files?